topofileformats/asd.py

```diff
@@ -176,31 +183,25 @@ def load_asd(file_path: Path | str, channel: str):
             header_dict = read_header_file_version_2(open_file)
         else:
             raise ValueError(
-                f"File version {file_version} unknown. Please add support if you\
-know how to decode this file version."
+                f"File version {file_version} unknown. Please add support if you "
+                "know how to decode this file version."
             )
-        print(header_dict)
+        logger.debug(f"header dict: \n{header_dict}")
 
         pixel_to_nanometre_scaling_factor_x = header_dict["x_nm"] / header_dict["x_pixels"]
         pixel_to_nanometre_scaling_factor_y = header_dict["y_nm"] / header_dict["y_pixels"]
         if pixel_to_nanometre_scaling_factor_x != pixel_to_nanometre_scaling_factor_y:
-            print(
-                f"WARNING: Resolution of image is different in x and y directions:\
-x: {pixel_to_nanometre_scaling_factor_x}\
- y: {pixel_to_nanometre_scaling_factor_y}"
+            logger.warning(
+                f"Resolution of image is different in x and y directions:"
+                f"x: {pixel_to_nanometre_scaling_factor_x}"
+                f"y: {pixel_to_nanometre_scaling_factor_y}"
             )
         pixel_to_nanometre_scaling_factor = pixel_to_nanometre_scaling_factor_x
 
         if channel == header_dict["channel1"]:
-            print(
-                f"Requested channel {channel} matches first channel in file: \
-{header_dict['channel1']}"
-            )
+            logger.info(f"Requested channel {channel} matches first channel in file: " f"{header_dict['channel1']}")
```

Comment:
If this all fits on a single line of < 120 characters should only need a single string.

```suggestion
            logger.info(f"Requested channel {channel} matches first channel in file: {header_dict['channel1']}")
```

Ditto line 204.